src/index.ts

```diff
@@ -82,11 +82,14 @@ const typescript: PluginImpl<RPT2Options> = (options) =>
 		cache.done();
 	}
 
+	const hasTransformersOrSourceMapCallback = options?.sourceMapCallback || options?.transformers?.length;
```

Comment:
```suggestion
	const hasFunctions = options?.sourceMapCallback || options?.transformers?.length;
```

Shorter and more durable name as more function options could be added like these two.
`hasFunctions` is also more meaningful in that sense, as to why these two options are special. Should probably add a comment above this stating that function scope can't be properly hashed, and linking to the `object-hash` issue